### ISSUE TITLE
Video tutorial on migrating to CEP-78

### ISSUE THREAD

**Issue by RitaMAllenCA** [casper-network/docs#828]
Create a video and we will review and discuss where to publish it.  Work with Karan on written tutorial.

**Comment by ipopescu** [comment]
Written tutorial submitted for review. The video and video editing would take some additional time.

**Comment by ipopescu** [comment]
I am waiting on a review before starting the recording.

**Comment by ipopescu** [comment]
The video is ready and will be sent to the Association for their review and publishing on youtube. I have created a new card to capture this work [here](https://github.com/casper-network/docs/issues/857).
